### PR TITLE
Migration: Sonification

### PR DESCRIPTION
**Pull request by marvin19**

### CODE REVIEW

File: ts/Extensions/Sonification/ChartSonify.ts [hunk 1]
```diff
@@ -825,8 +828,8 @@ namespace ChartSonify {
     }
 
     export interface ChartSonificationStateObject {
-        currentlyPlayingPoint?: Highcharts.SonifyablePoint;
-        timeline?: Highcharts.Timeline;
+        currentlyPlayingPoint?: Sonification.SonifyablePoint;
```
Comment (bre1470) [inline ts/Extensions/Sonification/ChartSonify.ts:831]: `Sonification.SonifyablePoint` should be integrated into `PointSonify.Composition`. Then you can delete the custom `this` scope of the `PointSonify.Composition` functions.

File: ts/Extensions/Sonification/PointSonify.ts [hunk 2]
```diff
@@ -74,9 +74,9 @@ namespace PointSonify {
...
 
     export declare class Composition extends Point {
-        pointCancelSonify(this: Highcharts.SonifyablePoint, fadeOut?: boolean): void;
+        pointCancelSonify(this: Sonification.SonifyablePoint, fadeOut?: boolean): void;
         pointSonify(
-            this: Highcharts.SonifyablePoint,
+            this: Sonification.SonifyablePoint,
             options: Options
         ): void;
     }
```
Comment (bre1470) [inline ts/Extensions/Sonification/PointSonify.ts:82]: That the spot, were it should get integrated.

**Review by bre1470 — CHANGES_REQUESTED** [review]
Nice job! There is one issue with the point typing.

**Review by bre1470 — APPROVED** [review]
Thank you! Looks good. 👍

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**